SVG.flood/fillImage.svg.flt

```diff
@@ -1,5 +1,5 @@
 <% if layer.type.name == "shape" %>
-<use href="#<%= layer.fillImage %>">
+<image xlink:href="<%= imageHref(image: layer.fillImage.image) %>" x="0" y="0">
```

Comment:
can we use `href` instead of `xlink:href`

Reply:
yep